libs/pyrt_component/pyrt/__init__.py

```diff
@@ -0,0 +1,12 @@
+from typing import TypeVar, Generic, Union, Optional, Protocol, Tuple, List, Any, Self
```

Comment:
Everything in this dir is generated from the wit file. Will remove this dir and generated when needed at the end of PR.